<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorGeneralPage</class>
 <widget class="QWidget" name="EditorGeneralPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>593</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure general editor settings&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line2">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Tabs &amp;&amp; Indentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="TextLabel13_3">
          <property name="text">
           <string>Tab width:</string>
          </property>
          <property name="buddy">
           <cstring>tabwidthSlider</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="tabwidthSlider">
          <property name="toolTip">
           <string>Move to set the tab width.</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLCDNumber" name="tabwidthLCD">
          <property name="toolTip">
           <string>Displays the selected tab width.</string>
          </property>
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="value" stdset="0">
           <double>4.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="TextLabel13_2_3">
          <property name="text">
           <string>Indentation width:</string>
          </property>
          <property name="buddy">
           <cstring>indentwidthSlider</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="indentwidthSlider">
          <property name="toolTip">
           <string>Move to set the indentation width.</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLCDNumber" name="indentwidthLCD">
          <property name="toolTip">
           <string>Displays the selected indentation width.</string>
          </property>
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="value" stdset="0">
           <double>4.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="autoindentCheckBox">
          <property name="toolTip">
           <string>Select whether autoindentation shall be enabled</string>
          </property>
          <property name="text">
           <string>Auto indentation</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="tabforindentationCheckBox">
          <property name="toolTip">
           <string>Select whether tab characters are used for indentations.</string>
          </property>
          <property name="text">
           <string>Use tabs for indentations</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="tabindentsCheckBox">
          <property name="toolTip">
           <string>Select whether pressing the tab key indents.</string>
          </property>
          <property name="text">
           <string>Tab key indents</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="converttabsCheckBox">
          <property name="toolTip">
           <string>Select whether tabs shall be converted upon opening the file</string>
          </property>
          <property name="text">
           <string>Convert tabs upon open</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Comments</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="comment0CheckBox">
        <property name="toolTip">
         <string>Select to insert the comment sign at column 0</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;Insert comment at column 0&lt;/b&gt;&lt;p&gt;Select to insert the comment sign at column 0. Otherwise, the comment sign is inserted at the first non-whitespace position.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Insert comment at column 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Virtual Space</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Virtual space is the space after the last character of a line. It is not allocated unless some text is entered or copied into it. Usage of virtual space can be configured with these selections.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="vsSelectionCheckBox">
        <property name="toolTip">
         <string>Select to enable a rectangular selection to extend into virtual space</string>
        </property>
        <property name="text">
         <string>Selection may access virtual space</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="vsUserCheckBox">
        <property name="toolTip">
         <string>Select to allow the cursor to be moved into virtual space</string>
        </property>
        <property name="text">
         <string>Cursor can move into virtual space</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>535</width>
       <height>101</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabwidthSlider</tabstop>
  <tabstop>indentwidthSlider</tabstop>
  <tabstop>autoindentCheckBox</tabstop>
  <tabstop>tabindentsCheckBox</tabstop>
  <tabstop>tabforindentationCheckBox</tabstop>
  <tabstop>converttabsCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>tabwidthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tabwidthLCD</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>indentwidthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>indentwidthLCD</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
